src/css/styles/_c-btn.scss

```diff
@@ -0,0 +1,38 @@
+.c-btn {
+  display: inline-block;
+  padding: 1.2rem 4rem;
+  font-size: 1.5rem;
+  font-weight: bold;
+  color: $col-bg;
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  letter-spacing: 0.1em;
+  background-color: $col-blue;
+  border: 0;
+  border-radius: 2rem;
+  transition-duration: 0.3s ease-in-out;
```

Comment:
IMHO we should split `transition-duration` to:

```css
  transition-timing-function: ease-in-out;
  transition-duration: 0.3s;
```

- [ ] transition-timing-function

Ref:
* https://www.w3schools.com/csSref/css3_pr_transition-duration.asp
* https://developer.mozilla.org/en-US/docs/Web/CSS/transition-duration

Reply:
Is there a reason not to use a shorthand like this?

https://developer.mozilla.org/en-US/docs/Web/CSS/transition

Reply:
We could use shorthand for `transition` but not for `transition-duration`